sound/soc/sof/ipc4-topology.c

```diff
@@ -2413,9 +2415,15 @@ static int sof_ipc4_widget_setup(struct snd_sof_dev *sdev, struct snd_sof_widget
 
 		msg->extension &= ~SOF_IPC4_MOD_EXT_PPL_ID_MASK;
 		msg->extension |= SOF_IPC4_MOD_EXT_PPL_ID(pipe_widget->instance_id);
+
+		dev_dbg(sdev->dev, "Create widget %s (pipe %d) - ID %d, instance %d, core %d\n",
+			swidget->widget->name, swidget->pipeline_id, module_id,
+			swidget->instance_id, swidget->core);
+	} else {
+		dev_dbg(sdev->dev, "Create widget %s (pipe %d) - instance %d, core %d\n",
```

Comment:
just a small suggestion to change this print to say "Create pipeline ..." as we're splitting the prints for a scheduler and other widgets

Reply:
sorry，forget to push to GH, I focused on develop script yesterday.
script are based on Create pipeline already, see:
https://github.com/thesofproject/sof-test/pull/1122/commits/9d2f43747490ec678bfbd09a168d652b101dde3a#diff-2c6a3bab8fe14ea957dc741dbaec639927363d2adc518fb90f9ae44a5ccd8bafR194